web/src/components/GenericTextField.tsx

```diff
@@ -101,9 +101,9 @@ export const GenericTextField = (props: GenericTextFieldProps): ReactElement =>
     : "";
   return (
     <div
-      className={`${props.formInputWide && "form-input-wide"} ${
-        !props.formInputFull && !props.formInputWide && "form-input"
-      }  margin-top-2`}
+      className={`${props.formInputWide ? "form-input-wide " : ""}${
+        !props.formInputFull && !props.formInputWide ? "form-input " : ""
+      }margin-top-2`}
```

Comment:
why this change? I prefer the first version more, where the class strings are just the string, instead of relying on something to have a trailing space 

To me, I prefer:
```
const x = "some-class"
className=`{$x} margin-top-2`
```

over:

```
const x = "some-class "
className=`{$x}margin-top-2`
```